src/k8s-extension/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+1.0.2
+++++++++++++++++++
+* microsoft.azureml.kubernetes: Retrieve relay and service bus connection string when update the configuration protected settings of the extension.
```

Comment:
Can you please add this to `1.0.1` since we have not released 1.0.1 yet

Reply:
Fixed.